### PR TITLE
Add kube-rbac-proxy image to values.yaml images list

### LINKED ISSUE #207: Add kube-rbac-proxy image to values.yaml images list
I am trying to internalize all the docker images related to Keda to my private repository. Based on the installation instructions provided for HTTP add on, the docker images mentioned are for operator, scaler and interceptor. The image for kube-rbac-proxy is still being pulled from an external source and hence, my deployment is failing as I have a private K8s cluster with no outbound internet access.

It would be really helpful if we can add the kube-rbac-proxy image to the images section in values.yaml file so that we can set the value in the helm install command.

### PR DESCRIPTION
**Pull request by Giridhar23689**
_Provide a description of what has been changed_

Referenced the value of kube-rbac-proxy image and tag from values.yml file.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/charts/blob/master/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [x] A PR is opened to update KEDA core ([repo](https://github.com/kedacore/keda)) *(if applicable, ie. when deployment manifests are modified)*
- [x] README is updated with new configuration values *(if applicable)*

Fixes #207

### CODE REVIEW

File: http-add-on/values.yaml [hunk 1]
```diff
@@ -113,9 +113,13 @@ images:
...
+  # rbacproxytag is the image tag for kube rbac proxy image
+  rbacproxytag: v0.5.0
   # the operator image to use. the default is the official build
   operator: ghcr.io/kedacore/http-add-on-operator
   # the interceptor image to use. the default is the official build
   interceptor: ghcr.io/kedacore/http-add-on-interceptor
   # the scaler image to use. the default is the official build
   scaler: ghcr.io/kedacore/http-add-on-scaler
+  # the kube-rbac-proxy image to use
+  kuberbacproxy: gcr.io/kubebuilder/kube-rbac-proxy
```
Comment (tomkerkhove) [inline http-add-on/values.yaml:125]: I'd use this approach to not pollute the rest
```suggestion
  kubeRbacProxy:
    name: gcr.io/kubebuilder/kube-rbac-proxy
    tag: v0.5.0
```

Comment (Giridhar23689) [reply http-add-on/values.yaml]: Updated the deployment- operator.yaml based on the suggestion. Please review

**Review by tomkerkhove — APPROVED** [review]
LGTM - Thanks!

**Comment by tomkerkhove** [comment]
Can you fix the DCO issues please? You can learn more in our [contribution guide](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#i-didnt-sign-my-commit-now-what).

**Review by arschles — APPROVED** [review]

**Comment by arschles** [comment]
@Giridhar23689 you'll just need to fix the DCO issues that @tomkerkhove mentioned [above](https://github.com/kedacore/charts/pull/210#issuecomment-984422730). Can you please also update the HTTP Addon chart's README with the new configuration values?

File: http-add-on/README.md [hunk 2]
```diff
@@ -80,12 +80,15 @@ The command removes all the Kubernetes components associated with the chart and
 The following table lists the configurable parameters of the Promitor chart and
 their default values.
 
+
```
Comment (tomkerkhove) [inline http-add-on/README.md:83]:
```suggestion
```

**Review by tomkerkhove — APPROVED** [review]
LGTM, just a nit

**Comment by Giridhar23689** [comment]
I have tried to push my changes with DCO for past 3 hours through Git CMD and UI but not able to do it at all. In CMD I am getting 
remote: No anonymous write access.
fatal: Authentication failed for 'https://github.com/Giridhar23689/charts.git/'

Even though I have used the correct password and PAT.

Please help

**Comment by tomkerkhove** [comment]
This should help you: https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#i-didnt-sign-my-commit-now-what

For the CLI, make sure to log in so that you can push to your own repo - https://docs.github.com/en/get-started/quickstart/set-up-git#next-steps-authenticating-with-github-from-git

**Comment by tomkerkhove** [comment]
This should help you: https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#i-didnt-sign-my-commit-now-what

For the CLI, make sure to log in so that you can push to your own repo - https://docs.github.com/en/get-started/quickstart/set-up-git#next-steps-authenticating-with-github-from-git

**Comment by Giridhar23689** [comment]
I have tried all the steps in the articles that you have provided. I have installed Github CLI as well. But I still face the same error:

PS C:\Users> gh auth login
? What account do you want to log into? GitHub.com
? You're already logged into github.com. Do you want to re-authenticate? Yes
? What is your preferred protocol for Git operations? HTTPS
? Authenticate Git with your GitHub credentials? Yes
? How would you like to authenticate GitHub CLI? Login with a web browser

! First copy your one-time code: F6FB-E384
- Press Enter to open github.com in your browser...
✓ Authentication complete. Press Enter to continue...

- gh config set -h github.com git_protocol https
✓ Configured git protocol
✓ Logged in as Giridhar23689
PS C:\Users>
PS C:\Users> git push --force
remote: No anonymous write access.
fatal: Authentication failed for 'https://github.com/Giridhar23689/charts.git/'


Not sure what is going wrong

**Comment by tomkerkhove** [comment]
I'm afraid that it's best to ask the GitHub community, sorry :/

It looks like your local repo is not correctly linked or so.

**Comment by zroubalik** [comment]
@tomkerkhove @Giridhar23689 so why don't we set DCO to pass and when squashing the commits provide the message there, if @Giridhar23689 agrees:
`Signed-off-by:  Giridhar Sripada giridharsripada@gmail.com`

**Comment by tomkerkhove** [comment]
Sounds good to me.

**Comment by Giridhar23689** [comment]
Thank you so much for merging it.

**Comment by tomkerkhove** [comment]
Thank you!

**Pull request merged**